src/lib/core/usecases.py

```diff
@@ -2287,3 +2287,41 @@ def execute(self):
             classes, open(Path(self._download_path) / "classes.json", "w"), indent=4
         )
         self._response.data = self._download_path
+
+
+class CreateAnnotationClassesUseCase(BaseUseCase):
+
+    CHUNK_SIZE = 500
+
+    def __init__(
+        self,
+        response: Response,
+        service: SuerannotateServiceProvider,
+        annotation_classes_repo: BaseManageableRepository,
+        annotation_classes: list,
+        project: ProjectEntity,
+    ):
+        super().__init__(response)
+        self._service = service
+        self._annotation_classes_repo = annotation_classes_repo
+        self._annotation_classes = annotation_classes
+        self._project = project
+
+    def execute(self):
+        existing_annotation_classes = self._annotation_classes_repo.get_all()
+        existing_classes_name = [i.name for i in existing_annotation_classes]
+        unique_annotation_classes = []
+        for annotation_class in self._annotation_classes:
```

Comment:
```suggestion
        for annotation_class in self._annotation_classes:
            if annotation_class["name"] not in existing_classes_name:
                unique_annotation_classes.append(annotation_class))
```

Reply:
or shorter
unique_annotation_classes = list(filter(lambda elem: elem["name"] not in existing_classes_name, self._annotation_classes))